### PR TITLE
[IA-3735] change rstudio background var name

### PR DESCRIPTION
**Pull request by jdcanas**
Openning this PR to run tests against updated welder hash

### CODE REVIEW

**Comment by jdcanas** [comment]
jenkins retest, fiab-startt

**Comment by jdcanas** [comment]
jenkins retest

**Comment by jdcanas** [comment]
jenkins retest fiab build

**Comment by jdcanas** [comment]
jenkins retest

**Comment by jdcanas** [comment]
jenkins retest

**Comment by Qi77Qi** [comment]
jenkins retest

**Comment by jdcanas** [comment]
jenkins retest

**Comment by jdcanas** [comment]
jenkins retest, not any test failures just fiab-start

File: http/src/main/resources/reference.conf [hunk 1]
```diff
@@ -558,7 +558,7 @@ gke {
 image {
   welderGcrUri = "us.gcr.io/broad-dsp-gcr-public/welder-server"
   welderDockerHubUri = "broadinstitute/welder-server"
-  welderHash = "08b3633"
+  welderHash = "f00465f"
```
Comment (mlilynolting) [inline http/src/main/resources/reference.conf]: Does this correspond to some official production instance of Welder?

**Review by mlilynolting — APPROVED** [review]

File: http/src/main/resources/reference.conf (hunk 1 above)
Comment (mlilynolting) [reply http/src/main/resources/reference.conf]: git commit hash, nm!

**Review by Qi77Qi — APPROVED** [review]
@jdcanas I just went ahead updating the hash

**Comment by jdcanas** [comment]
jenkins retest

**Comment by Qi77Qi** [comment]
```
      org.broadinstitute.dsde.workbench.leonardo.RestError: message: Failed to create app terra-quality-04646f48/automation-test-app-asdsnpirz, status: 403 Forbidden body: {"source":"leonardo","message":"ron.weasley@quality.firecloud.org is unauthorized. If you have proper permissions to use the workspace, make sure you are also added to the billing account","statusCode":403,"exceptionClass":"class org.broadinstitute.dsde.workbench.leonardo.model.ForbiddenError","traceId":null}
```

jenkins retest

**Pull request merged**